lib/confluence/lone_omission.es6.js

```diff
@@ -93,36 +95,35 @@ foam.CLASS({
       documentation: `Compute lone omission value based on browsers released
           during grace period.`,
       code: function(comparedReleases, prevReleases, date, release) {
+        const props = this.CompatData.getAxiomsByClass(this.CompatProperty);
+        const otherBrowserProps = props
+            .filter(p => comparedReleases
+                .some(r => foam.util.equals(p.release, r)));
+        const thisBrowserReleases = comparedReleases.concat([release]);
+        const thisBrowserProps = props
+            .filter(p => thisBrowserReleases
+                .some(r => foam.util.equals(p.release, r)));
+        const E = this.ExpressionsSingleton.create();
+
         // Compute:
-        // "APIs never shipped in browser-of-interest shipped, but shipped by
+        // "APIs never shipped in browser-of-interest, but shipped by
         //     all other browsers throughout grace period".
-        //
-        // i.e.,
-        //
-        // INTERSECT(<interfaces of other browsers from grace period>)
-        //     \ UNION(<interfaces in past browser-of-interest releases>).
-        var comparedReleasesLessOne = Array.from(comparedReleases);
-        var first = comparedReleasesLessOne.shift();
-        return first.interfaces.dao.select(this.INTERSECT(
-            comparedReleasesLessOne.map(rls => rls.interfaces.dao),
-            this.SET_MINUS(
-                this.UNION.apply(
-                    this,
-                    prevReleases.concat([release])
-                        .map(rls => rls.interfaces.dao)),
-                this.COUNT())))
-            .then(count => {
-              this.loneOmissionDAO.put(
+        const query = this.AND.apply(this, otherBrowserProps
```

Comment:
Won't |thisBrowserProps| be a superset of |otherBrowserProps| because of 102? Maybe that line should be using |prevReleases| instead of |comparedReleases|?

Reply:
Yeah, some changes got lost during some git-user-failure on my part. Latest push should be more reasonable.

Reply:
Apologies for wasting your time reviewing the wrong code :man_facepalming: